tests/test.sh

```diff
@@ -776,6 +776,10 @@ make_test()
     check_log_message remoteice2 "Remote compilation completed with exit code 0"
     check_log_error remoteice2 "Remote compilation aborted with exit code"
     check_log_error remoteice2 "Remote compilation exited with exit code $expected_exit"
+    if [ -z "$SAN_FLAGS" ]; then
+        # The setrlimit call is only made (and logged) in non-sanitizer builds.
+        check_log_message remoteice1 "Compile job memory limit set to "
+    fi
```

Comment:
I think this part should be removed, as it actually doesn't do anything in practice (it checks for a presence of a "random" log message, but it doesn't really test or verify anything). Moreover, whether the message is logged or not is not controlled by $SAN_FLAGS, but by a configure check, so it's also incorrect.

Reply:
It indirectly confirms that there was a memory limit set (assuming that SAN_FLAGS is set whenever making sanitizer builds).  But I agree that it's not vital- removed.